packages/angular-table/src/index.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  Directive,
+  Input,
+  OnInit,
+  TemplateRef,
+  ViewContainerRef,
+} from '@angular/core'
+import {
+  RowData,
+  TableOptions,
+  TableOptionsResolved,
+  createTable,
+} from '@tanstack/table-core'
+
+export * from '@tanstack/table-core'
+
+@Directive({
+  selector: '[flexRender]',
+  standalone: true,
+})
+export class FlexRenderDirective implements OnInit {
+  private _flexRender: any
+
+  /** properties to render */
+  private _flexRenderProps: any
+
+  @Input({ required: true })
```

Comment:
Hi, all input/output should now use signal based [input](https://angular.io/guide/signal-inputs)/[model](https://angular.io/guide/model-inputs)/[output](https://github.com/angular/angular/pull/54650).

Reply:
Yeah, that was a question for me. If using Angular 17 as the minimum peerDep, should this use signals? Or should we support much lower versions of Angular?

Reply:
I am assuming initial version will be experimental like Angular tanstack query so its safe to use latest angular as minimum version I think 17.3 (currently in rc) could be used as minimum if output api is needed, and later it can be changed to 18 too, if this adapter stays as experimental until May 24.